cmd/podman/rmi.go

```diff
@@ -58,11 +57,11 @@ func rmiCmd(c *cli.Context) error {
 		return err
 	}
 	removeAll := c.Bool("all")
-	runtime, err := libpodruntime.GetRuntime(c)
+	localRuntime, err := adapter.GetRuntime(c)
```

Comment:
Isn't runtime more descriptive here than localRuntime? Or am I wrong that adapter could return remoteRuntime?

Reply:
yes, but no solid, agreeable name has been determined.  lets work this out as we go.